### PR TITLE
ROX-9652: Version port to Postgres

### PR DESCRIPTION
**Pull request by connorgorman**
## Description

Initial move of Version to Postgres.  Plan is for it to work mostly the same as it does now so a straight port is the first step.

## Checklist
- [x] Investigated and inspected CI test results
- [x] Unit test and regression tests added
- ~~[ ] Evaluated and added CHANGELOG entry if required~~
- ~~[ ] Determined and documented upgrade steps~~
- ~~[ ] Documented user facing changes (create PR based on [openshift/openshift-docs]~~(https://github.com/openshift/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))

If any of these don't apply, please comment below.

## Testing Performed

CI failures are unrelated and already have tickets.

### CODE REVIEW

**Comment by connorgorman** [comment]
Okay this one is unmergeable until we figure out the migrator

**Comment by dashrews78** [comment]
@connorgorman I checked with @c-du and she currently anticipates version working basically the same as it does before postgres where we simply use a sequence number.  Obviously that may evolve, but I think a simple port, will suffice initially.

File: central/main.go [hunk 1]
```diff
@@ -132,6 +132,7 @@ import (
 	"github.com/stackrox/rox/central/ui"
 	userService "github.com/stackrox/rox/central/user/service"
 	"github.com/stackrox/rox/central/version"
+	"github.com/stackrox/rox/central/version/store"
```
Comment (c-du) [inline central/main.go:135]: Since this is central/main, it would be better to call it like vStore.

**Review by c-du — APPROVED** [review]
With a minor comment.

**Pull request merged**